### PR TITLE
Abby Tiffany Sprint

### PR DESCRIPTION
**Pull request by atiffany**
Sprint Challenge Completed

### CODE REVIEW

**Comment by adamlower** [comment]
Hey Abby, I just did an initial review of your code and everything looks good so far. Keep up the great work and I'm looking forward to a PR with the finished result.

**Comment by adamlower** [comment]
Hey Abby, it looks like you have a really great command of this material. The code works without any issues and your style and spacing are consistent. Keep up the great work!

File: project/ls8/cpu.js [hunk 1]
```diff
@@ -4,19 +4,30 @@
...
+const JMP  = 0b01010000;
+const JNE  = 0b01010010;
+const LDI  = 0b10011001; 
+const MUL  = 0b10101010;
+const NOP  = 0b00000000;
+const POP  = 0b01001100;
+const PRN  = 0b01000011;
+const PUSH = 0b01001101;
+const RET  = 0b00001001;
+const SP = 7; //according to Google Stack Pointer needs to point to Reg7
```
Comment (adamlower) [inline project/ls8/cpu.js]: For consistency I believe it'd be better to use a HEX value here instead, like 0x07.

**Comment by atiffany** [comment]
Should I use hex or should I use binary since my other declarations were
binary?

**Comment by adamlower** [comment]
Opcodes are represented in binary because they are instructions for the CPU. The stack pointer is just used to keep track of the call stack, specifically pointing to the address at the top of the stack. We represent those physical addresses that it points to in hex.

**Comment by atiffany** [comment]
That is exactly answer and info I've been looking for! Thank you!

**Pull request closed without merging**